server/modules/instantiation/jobs.js

```diff
@@ -303,16 +305,19 @@ module.exports = function (config, portProvider, interpolationHelper, buildInsta
             var { buildInstance } = job;
```

Comment:
the formatting could be simplified in the whole file, let's stick to what I've changed here, jobs.js will be refactored to smaller classes anyway so then we can improve the formatting a bit